src/utils/send-it.js

```diff
@@ -1,8 +1,28 @@
 const fs = require('fs');
+const inquirer = require('inquirer');
 const s3 = require('../s3');
 
 const uploadsFolder = 'uploads/';
-const targetFolder = 'staging/';
+let targetFolder = process.argv.slice(2)[0];
```

Comment:
It's hard to follow `targetFolder` since it can get reassigned in the code below, and it is also used in the `uploadUniqueFilesToS3` function.

Let's change the `let` to `const` and rename to:
```javascript
const targetFolderArg = process.argv.slice(2)[0];
```

This will be clearer that this variable is from a command line argument.

Following up, we can now change:
```javascript
targetFolder = userAnswer.folderDestination;
```
to this:
```javascript
const targetFolderAnswer = userAnswer.folderDestination;
```

Then we can update `uploadUniqueFilesToS3`'s function signature to:
```javascript
const uploadUniqueFilesToS3 = async (bucketFolder, localRawFileNames, targetFolder)
```
Now you can pass in your new variables into `uploadUniqueFilesToS3`.

That way, we can avoid modifying global variables, and make the variable usage easier to follow. Let me know what you think!